h2o-py/h2o/frame.py

```diff
@@ -30,9 +32,8 @@
 from h2o.utils.shared_utils import (_handle_numpy_array, _handle_pandas_data_frame, _handle_python_dicts,
                                     _handle_python_lists, _is_list, _is_str_list, _py_tmp_key, _quoted,
                                     can_use_pandas, quote)
-from h2o.utils.typechecks import I, U, assert_is_type, assert_satisfies, is_type, numpy_ndarray, pandas_dataframe
-
-import requests
+from h2o.utils.typechecks import (assert_is_type, assert_satisfies, I, is_type, numeric, numpy_ndarray,
+                                  pandas_dataframe, U)
```

Comment:
U = union of types, I = intersection of types (really it ought to have been "∩", but you can't have a variable name like that)